front-office/search.md

```diff
@@ -21,3 +21,12 @@ If the visitor reaches a page that contains no product, the page should display
 `Stay tuned! More products will be shown here as they are added.` in `Shop.Theme.Catalog`
 
 A search bar is available right below to allow visitors to try another query: `Search our catalog` in `Shop.Theme.Global`.
+
+## 404 pages 
+
+If the visitor reaches an available page, the page should display the following message:
```

Comment:
```suggestion
If the visitor reaches an unavailable page, the page should display the following message:
```

Maybe I'm wrong, but shouldn't it be "unavailable page" for the 404 behaviour ?

Reply:
You are completely right, thanks!